### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by NataliyaKoval**

### CODE REVIEW

File: src/app/components/performance/performance.component.scss [hunk 1]
```diff
@@ -140,13 +140,13 @@
   }
 }
 
-.link {
+::ng-deep .ngx-gallery-thumbnail.ngx-gallery-active {
```
Comment (yana-tsylinska) [inline src/app/components/performance/performance.component.scss:143]: :host ::ng-deep

File: src/app/components/performance/performance.component.scss [hunk 2]
```diff
@@ -140,13 +140,13 @@
...
+  border: none !important;
+}
+
+.details::ng-deep a {
```
Comment (yana-tsylinska) [inline src/app/components/performance/performance.component.scss:147]: :host

**Review by yana-tsylinska — APPROVED** [review]

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]
add :host

File: src/app/components/performance/performance.component.scss (hunk 1 above)
Comment (NataliyaKoval) [reply src/app/components/performance/performance.component.scss]: done

File: src/app/components/performance/performance.component.scss (hunk 2 above)
Comment (NataliyaKoval) [reply src/app/components/performance/performance.component.scss]: done

**Review by yana-tsylinska — APPROVED** [review]

**Pull request merged**